cmd/cli/task_config/ask_plan.go

```diff
@@ -0,0 +1,195 @@
+package task_config
+
+import (
+	"errors"
+	"fmt"
+	"math"
+	"math/big"
+	"strconv"
+	"strings"
+	"time"
+
+	ds "github.com/c2h5oh/datasize"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/params"
+	"github.com/jinzhu/configor"
+	"github.com/sonm-io/core/proto"
+)
+
+const (
+	minRamSize  uint64 = 4 * 1024 * 1024
+	minCPUCount        = 0.01
+)
+
+type DataSize struct {
+	size uint64
+}
+
+func (d *DataSize) Bytes() uint64 {
```

Comment:
II don't like this at all. Why can not you make a custom unmarshaller for protobuf struct?

Reply:
Probably we can, but there will be an unmaintainable piece of shit instead of well-bordered human-readable representation and machine-friendly protobufs.